java/src/main/java/ai/onnxruntime/OrtSession.java

```diff
@@ -548,6 +548,26 @@ public void addNuphar(boolean allowUnalignedBuffers, String settings) throws Ort
       addNuphar(OnnxRuntime.ortApiHandle, nativeHandle, allowUnalignedBuffers ? 1 : 0, settings);
     }
 
+    /**
+     * Adds DirectML as an execution backend.
+     *
+     * @param deviceNum The id of the DirectML device.
+     * @throws OrtException If there was an error in native code.
+     */
+    public void addDirectML(int deviceNum) throws OrtException {
```

Comment:
nit: I would call this device id to be consistent with the C API. Same for documentation above.

Reply:
Done.